### DISCUSSION TITLE
Welcome to ReinforcementLearning.jl Discussions!

### DISCUSSION THREAD

**Discussion by findmyway** [JuliaReinforcementLearning/ReinforcementLearning.jl#164] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by oysteinsolheim** [comment]
Hi there! I am using Julia and RL in application to power system reliability research. Have been using Julia for a year and a half after coming from the Python world. Not sure I'm going back 😊

Rgds Øystein
